### ISSUE TITLE
Move `analysers` to their own repo?

### ISSUE THREAD

**Issue by geohacker** [mapbox/tile-reduce-peer-review#10]
@Rub21 @batpad @maning @mikelmaron - should we move these analysers to their own repos? I think they are largely independent of how peer review processor works and could be made modular. Thoughts?

**Comment by Rub21** [comment]
@geohacker I am agree with the proposal, even we can do each analyzers can work independent from peer-review.

**Comment by maning** [comment]
Agree. Either add it in https://github.com/mapbox/tile-reduce-processors or a separate repo.
